### PR TITLE
Reworked angular slider topic

### PR DESCRIPTION
**Pull request by sboykova**
Issue #46

### CODE REVIEW

File: en/components/slider.md [hunk 1]
```diff
@@ -5,82 +5,244 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 ---
 
 ##Slider
-<p class="highlight">Configure a slider or range slider with the Ignite UI for Angular Slider control. Users can select a value in a given range, or a range of values. The range can be defined as continuous or stepped.</p>
+<p class="highlight">The Ignite UI for Angular Slider component allows selection in a given range by moving the thumb along the track. The track can be defined as continuous or stepped and you can choose between single and range slider type.</p>
```
Comment (zdrawku) [inline en/components/slider.md]: .. and you can choose between single and range slider type**s**

**Comment by ig-ealbert** [comment]
https://github.com/IgniteUI/igniteui-docfx/pull/89

**Comment by ig-ealbert** [comment]
@kdinev What should I do with my pull request #89 now?  I created it as a pull request into Svetla's branch, which has now been merged into master.

**Comment by kdinev** [comment]
@ig-ealbert I commented inside the PR.

**Pull request merged**